install.sh

```diff
@@ -26,8 +26,9 @@ The installer for Scarb
 Usage: install.sh [OPTIONS]
 
 Options:
-    -v        Specify Scarb _requested_version to install
-    -h        Print help information
+    -h | --help             Print help information
+    -p | --no-modify-path   Skip PATH variable modification
+    -v | --version          Specify Scarb _requested_version to install
```

Comment:
```suggestion
  -p, --no-modify-path   Skip PATH variable modification
  -h, --help             Print help
  -v, --version          Specify Scarb version to install
```

let's stick with `clap` style:
<img width="1082" alt="image" src="https://github.com/software-mansion/scarb/assets/3450050/be714d5c-19b7-4e02-9fb9-69e17176f906">